### ISSUE TITLE
Tag for v2.8.3

### ISSUE THREAD

**Issue by icp1994** [rrthomas/enchant#401]
Is the version 2.8.3 intended to be in testing phase? Since NEWS says it's released on Oct 6 but missing a tag/release.

**Comment by rrthomas** [comment]
Yes, it's still being tested. I'll leave this issue open for anyone else confused.

**Comment by rrthomas** [comment]
2.8.3 is now released; closing.
